src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -107,7 +107,17 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
 
         [Option(
             "baseline",
-            HelpText = "A Sarif file to be used as baseline")]
+            HelpText = "A SARIF file to be used as baseline.")]
         public string BaselineSarifFile { get; set; }
+
+        [Option(
+            "automationId",
+            HelpText = "An id that will be used in the output SARIF and show in the 'automationDetails.id' under 'run'.")]
```

Comment:
>An id that will be used in the output SARIF and show in the 'automationDetails.id' under 'run'. [](http://example.com/codeflow?start=24&length=95)

"An id that will be persisted to the 'Run.AutomationDetails.Id' property. See section '3.17.3' of the SARIF specification."

Reply:
added.
thanks!